components/azure-iot/devdoc/platform_openssl_compact_requirements.md

```diff
@@ -0,0 +1,89 @@
+platform_openssl_compact
+=============
+
+## Overview
+
+**platform_openssl_compact** implements the **platform** adapter for the Azure IoT C SDK. This implementation
+uses a version of the **tlsio** adapter for OpenSSL which is aimed at embedded applications.
+
+Unlike non-micro systems such as Windows and Linux, embedded systems typically
+require explicit NTP initialization, so platform_openssl_compact performs NTP
+initialization and deinitialization using the interfaces in `azure-c-shared-utility/inc/azure_c_shared_utility/sntp.h`.
+
+## References
+[sntp.h](https://github.com/Azure/azure-c-shared-utility/tree/master/inc/azure_c_shared_utility/sntp.h)
+
+###   Standard
+
+**SRS_PLATFORM_OPENSSL_COMPACT_30_001: [** The platform_openssl_compact shall implement the interface provided in the `platfom.h`.
```

Comment:
probably a typo here, 'platfom' -> 'platform'. Also, is there a document which explains what these identifiers (SRS_PLATFORM_OPENSSL_COMPACT_30_001) mean?